common/changes/@uifabric/example-app-base/jspurlin-ComboBoxScrollIntoViewFix_--2090-_2017-06-29-13-27.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
```

Comment:
-b BRANCH, --target-branch BRANCH
                        If this parameter is specified, compare current 
                        branch with the target branch to get changes. If this 
                        parameter is not specified, the current branch is 
                        compared against the "master" branch.